### ISSUE TITLE
Found Element By Location

### ISSUE THREAD

**Issue by simpleMemory** [go-rod/rod#121]
Some sites use anti-crawler technology on the front，like Confusing fonts，
In this case, it is difficult to access the elements through the usual positioning methods。

The above problem can be solved by providing a positioning method based on the coordinate position of the elements

**Comment by ysmood** [comment]
Make sense to me.

You can already get it with rod, example code:

```go
proto.DOMGetNodeForLocation{X: 100, Y: 100, IncludeUserAgentShadowDOM: true}.Call(page)
```

Can you help us to create a PR to add this feature to rod? It should be something like `page.ElementByPos(100, 100)`.

**Comment by ysmood** [comment]
resolved in 833565d0a02e3f9d038d804070ea3b37af858852

**Comment by simpleMemory** [comment]
sorry，I didn't check back in time.
